holidays/countries/hungary.py

```
@@ -117,6 +117,25 @@ class HungaryStaticHolidays:
 
     References:
+        * [1991]() 7/1990. (XII. 27.) MüM rendelet - az 1991. évi munkaszüneti napok körüli munkarendről
+          The data is taken from a payed online database: https://uj.jogtar.hu
+        * [1992](https://web.archive.org/web/20250526085935/https://jogkodex.hu/jsz/1992_3_mum_rendelet_5937748)
+        * [1993](https://web.archive.org/web/20250526090343/https://jogkodex.hu/jsz/1992_7_mum_rendelet_7815697)
+        * [1994](https://web.archive.org/web/20250526090541/https://jogkodex.hu/jsz/1993_3_mum_rendelet_3120363)
+        * [1997](https://web.archive.org/web/20250526090817/https://jogkodex.hu/jsz/1996_11_mum_rendelet_3554324)
+        * [1998](https://web.archive.org/web/20250526091318/https://jogkodex.hu/jsz/1997_18_mum_rendelet_7493439)
+        * [1999](https://web.archive.org/web/20250526093916/https://jogkodex.hu/jsz/1998_3_szcsm_rendelet_7336830)
+        * [2001](https://web.archive.org/web/20250526094003/https://jogkodex.hu/jsz/2000_43_gm_rendelet_9845985)
+          The online data for 2001 is incomplete. Probably due to error in digitalization. 
```

Ruff (0.11.9)

Trailing whitespace

(W291)